test_files/labels/bad_lid_label.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Product_Observational
+  xmlns="http://pds.nasa.gov/pds4/pds/v1"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="
+  http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1B00.xsd
+  http://pds.nasa.gov/pds4/disp/v1 https://pds.nasa.gov/pds4/disp/v1/PDS4_DISP_1B00.xsd
+  http://pds.nasa.gov/pds4/mission/cassini/v1 https://pds.nasa.gov/pds4/mission/cassini/v1/PDS4_CASSINI_1B00_1300.xsd">
+    <Identification_Area>
+        <version_id>1.0</version_id>
+        <title>Cassini ISS Image 1455200455n.img</title>
+        <information_model_version>1.11.0.0</information_model_version>
+    </Identification_Area>
+    <Observing_System>
+        <name>Cassini Orbiter Imaging Science Subsystem</name>
+        <Observing_System_Component>
+            <name>Cassini Orbiter</name>
+            <type>Spacecraft</type>
+            <Internal_Reference>
+                <lid_reference>urn:nasa:pds:context:instrument_host:spacecraft.co</lid_reference>
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/SETI/rms-pds4indextools/pull/38#issuecomment-2310596918>#91ec65</a><span> - 08/26/2024, 04:41 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The LID reference in the XML file is incomplete and does not follow the PDS4 standard format.
</div>
<div id="fix">
<b>Fix:</b> Update the LID reference to follow the correct PDS4 format, including all necessary components (urn:agency:archive:bundle:collection:product).
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -20,1 +20,1 @@
 -                <lid_reference>urn:nasa:pds:context:instrument_host:spacecraft.co</lid_reference>
 +                <lid_reference>urn:nasa:pds:context:instrument_host:spacecraft.cassini</lid_reference>
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged

Reply:
This is the lid reference for the Cassini Orbiter. See [here](https://pds.nasa.gov/datastandards/training/2017-agu/context-list-instrument_host.pdf)